webpack/assets/javascripts/react_app/components/common/charts/LineChart/LineChart.fixtures.js

```diff
@@ -0,0 +1,15 @@
+export const data = [
+  ['red', [5, 7, 9], '#AA4643'],
```

Comment:
I agree, this is awkward. It is because of how our [ChartService](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/services/charts/ChartService.js#L50) works and what data it expect. I wanted to avoid modifying the service as I would need to change data format for all the graphs, so I wrapped it with `LineChartService` and did some additional modifications on top of it - like the destructuring you mentioned below.

Also, c3 expects [data columns](https://c3js.org/samples/simple_multiple.html) as arrays where first item is a label followed by values...

This is a comparison of what data format I pass to `LineChartService` and what chart config I get back:
![data-cmp](https://user-images.githubusercontent.com/2664090/63841659-7c9be100-c983-11e9-91f8-100f4e60df43.png)